apartmenttherapy/oxygen-lib-ui
# remove original width and height from components

#### What does this PR do & why?
- remove `original width` and `original height` from components

#### How can this be manually tested?
- use pr and local gql: https://github.com/apartmenttherapy/oxygen-gql/pull/81
- go to any of the mock pages listed below:
   - http://localhost:8080/
  - http://localhost:8080/collection/bundle-samples
  - http://localhost:8080/garlic-steak-and-potatoes-258348
  - http://localhost:8080/food
  - http://localhost:8080/food/breakfast
  - http://localhost:8080/recipes
  - http://localhost:8080/test-embeds-261284-261284
- see that all images load properly

#### Any questions or concerns about this change?

_(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)_

#### Deployment Notes

_(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)_

#### Link to story card and other documentation

#### GIF

#### Have you remembered to...?
